package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "@types/node": "^18.16.0",
```

Comment:
Needed to resolve the BigInt references in the code.  Hypersync SDK doesn't need it because it pulls in `@types/jsonwebtoken` which has a dependency on `@types/node`.